packages/payload/src/admin/components/views/LivePreview/Toolbar/Controls/index.tsx

```diff
@@ -13,27 +14,53 @@ const baseClass = 'live-preview-toolbar-controls'
 export const ToolbarControls: React.FC<EditViewProps> = () => {
   const { breakpoint, breakpoints, setBreakpoint, setPreviewWindowType, setZoom, url, zoom } =
     useLivePreviewContext()
+  const zoomOptions = [50,75,100,125,150,200]
```

Comment:
Can you move this and the next two variables to outside the component definition? Before line 14. This is because these variables are defined once and will never change.